apps/email/test/marionette/local_drafts_test.js

```diff
@@ -1,3 +1,5 @@
+'use strict';
+
```

Comment:
For me to run this test, Firefox complained that `const` was not allowed with that setup. So locally I changed the `const` variables in this file to `var` then it ran, so I suggest doing that.